pipelines/folioRancher/folioQualityGates/Jenkinsfile

```diff
@@ -128,17 +150,32 @@ ansiColor('xterm') {
           ]
           folioCypressFlow(cypressParams)
         }
+        branches['Karate'] = { ->
```

Comment:
this requires okapi to be in cluster mode, I guess...

Reply:
It's separate thing, not included in this scope